### ISSUE TITLE
In 02-manual-operations.md Compute Engine API needs to be enabled

### ISSUE THREAD

**Issue by tonykay** [Artemmkin/infrastructure-as-code-tutorial#5]
In 02-manual-operations.md the instruction:

> glcoud compute instance create

will/may fail with a message to enable "enable Compute Engine API". The error message also gives the fix which is to follow the link and enable the API: `https://console.developers.google.com/apis/api/compute.googleapis.com/overview?project=xxxxxxxxxxxx then retry`

**Comment by Artemmkin** [comment]
yep, I'm aware of it. If you have any suggestions on what we can add to the tutorial to avoid this error from happening or make it less confusing for people, that would be awesome. I will give it a thought too. Although, I think this error is not so critical as it gives you a solution on how to fix it.

**Comment by tonykay** [comment]
I agree, if I see a fix I'll let you know. More familiar with AWS than GCP.

**Comment by j-i-m-s-t-e-r** [comment]
You also need billing setup (a credit card) even on Free Tier now if you try to create compute resource.
